docs/applications/intro-to-vim-customization.md

```diff
@@ -0,0 +1,304 @@
+vnoremap <Space> zf
+
+" Automatically save and load folds
+autocmd BufWinLeave *.* mkview
+autocmd BufWinEnter *.* silent loadview
+~~~
+
+2. Each customization in the **.vimrc** file is paired with an explanation specifying its impact on the Vim editor. You may want to read through line by line and disable any customizations you feel are not necessary by commenting them out (comments are defined by the "`"`" character).
+
+### Organize The *.vimrc* File With Folds
+
+The **.vimrc** file can quickly become cluttered and disorganized as more and more customizations are added. An excellent approach to file management is using Vim's *Fold* feature. 
+
+1. From the still active **.vimrc** file, navigate your cursor to the line that reads "Display options". Type the following sequence of commands in the order listed.
+
+- Enter visual line mode by holding the `<shift>` key and pressing `v`
+- Type `15j` to block select all of the display related customizations
+- Type `zf` to create the fold
+
+2. This collapses all of the display related options neatly into a single row, with *Display options* visible for later identification. Normally, any created Folds would be deleted when Vim is closed, even if the file was saved. However, the text block added at the very end of the **.vimrc** file automatically saves and reproduces the state of any folds made in a file. Save and close the **.vimrc** file when you are finished editing. Your added customizations will take effect the next time Vim is opened. More commands to interact with folds are detailed below.
+
+- `zo`: Open the fold
+- `zc`: Close the fold
+- `zf`: Create a new fold
+- `zd`: Delete a fold
+
+## Integrate Plugins
+
+Plugins are a powerful way to customize your Vim instance. They can exponentially expand Vim's capabilities and allow the user to perform tasks which would traditionally require specialized software. This entire guide was written, previewed, and uploaded to Github using Vim.
+
+### Install The Vim-Plug Plugin Manager
+
+The most effective way to install and manage plugins requires the use of a plugin management tool. Vim-Plug will be used here, but there are a handful of others to choose from.
+
+1. Install *curl*.
+
+    **Fedora/RHEL based**
+
+        sudo yum install curl
+
+    **Debian based**
+
+        sudo apt install curl
+
+    **Arch Linux**
+
+        sudo pacman -Syy curl
+
+2. Create the installation directories, download, and install VimPlug from Github.
+
+        sudo curl -fLo ~/.vim/autoload/plug.vim --create-dirs https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
+
+### Install Your First Plugin With Vim-Plug
+
+Installing plugins with Vim-Plug is very simple. Once you identify a plugin you would like to add to Vim, it is as easy as adding a line of code to your **.vimrc** file. Below, installing the NERDTree file system explorer plugin will be demonstrated. All other plugins are installed in exactly the same manner.
```

Comment:
Try to avoid additional stylistic comments (e.g. `Installing plugins with Vim-Plug is very simple.`)